src/components/ItaliaTheme/View/Commons/BandoStatus.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * Importante: (06/11/2023)
+ * Component creato per essere possibile customizzare in modo più efficiente le traduzioni per lo status dei bandi.
+ * Deve essere cancellato dopo l'aggiornamento di Volto
+ */
+
+import { defineMessages, useIntl } from 'react-intl';
+
+const messages = defineMessages({
+  open: {
+    id: 'bando_open',
+    defaultMessage: 'attivo',
```

Comment:
i default message vanno mantenuti come erano scritti prima, con l'iniziale maiuscola

Reply:
@giuliaghisini Entrambi component usano tutto maiuscolo nella visualizzazione e il component "PageHeaderBando" è una frase e non una parola unica, sarebbe strano scrivere "Bando In corso", no? Ma credo che non ci sono problemi perché sono sempre tutto maiuscole.
 
<img width="356" alt="Screenshot 2023-11-07 at 09 35 49" src="https://github.com/RedTurtle/design-comuni-plone-theme/assets/60133113/ab95a544-d3ab-4e44-b53e-f0fdf35cebcc">
<img width="454" alt="Screenshot 2023-11-07 at 09 36 14" src="https://github.com/RedTurtle/design-comuni-plone-theme/assets/60133113/fec909a2-30fc-4f52-9d7e-ca162ed1063e">